src/lib.rs

```diff
@@ -1,11 +1,21 @@
 #![feature(plugin)]
+#![feature(test)]
 
 #![plugin(clippy)]
 
+#[cfg(test)]
+extern crate test;
+
+#[cfg(test)]
+#[macro_use]
+extern crate quickcheck;
+
+extern crate nalgebra;
+
+pub mod aabb;
 
 #[cfg(test)]
```

Comment:
In this case the test modules [will be part](https://doc.rust-lang.org/book/testing.html#the-tests-module) of the compiled library.